server/graphql/general/src/queries/plugin.rs

```diff
@@ -0,0 +1,21 @@
+use async_graphql::*;
+use graphql_core::ContextExt;
+use graphql_types::types::PluginNode;
+use service::plugin_files::PluginFileService;
+
+pub fn get_plugins(ctx: &Context<'_>) -> Result<Vec<PluginNode>> {
+    let settings = ctx.get_settings();
```

Comment:
We don't need authentication, right?

Reply:
I don't think so - my view was that the plugins are at the level of application architecture. maybe later we'd configure per store.. for now it's all or nothing